system/mp/test/mptest_mpi.cpp

```diff
@@ -0,0 +1,645 @@
+
+
```

Comment:
this file should be deleted and all test that work on ICommunicator's that are new should be moved into mptest.cpp
If there is anything specific to MPI can stay here.